flag.go

```diff
@@ -595,3 +596,27 @@ func withEnvHint(envVar, str string) string {
 	}
 	return str + envText
 }
+
+func stringifyFlag(f Flag) string {
+	fv := reflect.ValueOf(f)
+	placeholder, usage := unquoteUsage(fv.FieldByName("Usage").String())
```

Comment:
Never mind, that would mean this wouldn't address #257. Let's keep it as is -- this would be an advanced feature anyway.